src/commons_abstract_fevalues.jl

```diff
@@ -128,7 +137,7 @@ where ``u_i`` are the value of ``u`` in the nodes. For a vector valued function
 ``\\mathbf{u}(\\mathbf{x}) = \\sum\\limits_{i = 1}^n N_i (\\mathbf{x}) \\mathbf{u}_i`` where ``\\mathbf{u}_i`` are the
 nodal values of ``\\mathbf{u}``.
 """
-@inline function function_value{dim, T}(fe_v::AbstractFEValues{dim}, q_point::Int, u::Vector{T})
+function function_value{dim, T}(fe_v::AbstractFEValues{dim}, q_point::Int, u::Vector{T})
```

Comment:
this could be defined for the union of `FECellScalarValues` and `FEBoundaryScalarValues` later on.